genshin/models/genshin/chronicle/notes.py

```diff
@@ -73,6 +73,49 @@ def seconds(self) -> int:
         return self.timedata[3]
 
 
+class TaskReward(APIModel):
+    """Status of the Commission/Task.
+
+    status:
+        TaskRewardStatusTakenAward
+        TaskRewardStatusUnfinished
```

Comment:
I have no idea what this means. Is this like `typing.Literal["TaskRewardStatusTakenAward", "TaskRewardStatusUnfinished"]`?

Also please have the docstrings under the attributes, rather than on the class level.